app/src/main/java/com/gri/androidacademyworkshop/viewHolders/ActorsViewHolder.kt

```diff
@@ -0,0 +1,27 @@
+package com.gri.androidacademyworkshop.viewHolders
+
+import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.gri.androidacademyworkshop.R
+import com.gri.androidacademyworkshop.model.Actor
+
+class ActorsViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
```

Comment:
keep ViewHolders in same kt file with adapter when ViewHolder is not used in another adapter